packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -1030,6 +1038,35 @@ class RawScrollbar extends StatefulWidget {
   /// If null, defaults to Color(0x66BCBCBC).
   final Color? thumbColor;
 
+  /// The preferred smallest size the scrollbar can shrink to when the total
+  /// scrollable extent is large, the current visible viewport is small, and the
+  /// viewport is not overscrolled.
+  ///
+  /// The size of the scrollbar may shrink to a smaller size than [minThumbLength] to
+  /// fit in the available paint area. E.g., when [minThumbLength] is
+  /// `double.infinity`, it will not be respected if
+  /// [ScrollMetrics.viewportDimension] and [mainAxisMargin] are finite.
+  ///
+  /// Mustn't be null and the value has to be greater or equal to
+  /// [minOverscrollLength], which in turn is >= 0. Defaults to 18.0.
+  final double minThumbLength;
+
+  /// The preferred smallest size the scrollbar can shrink to when viewport is
```

Comment:
```suggestion
  /// The preferred smallest size the scrollbar thumb can shrink to when viewport is
```

We should clarify here and elsewhere that it is the scrollbar thumb. There is a track now too.